app/src/main/res/layout/layout3.xml

```diff
@@ -155,3 +155,4 @@
 
 
     </androidx.constraintlayout.widget.ConstraintLayout>
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
Added an end of line for the androidx.constraintlayout.widget.ConstraintLayout block